package.json

```diff
@@ -4,20 +4,38 @@
   "description": "REST API boilerplate",
   "main": "server.js",
   "dependencies": {
+    "Validator": "^1.0.4",
```

Comment:
Hum... while this makes it easier for the PHP users to adapt, the package doesn't seem that actively developed/maintained/used compare to 
https://github.com/ctavan/express-validator

https://www.npmjs.com/package/express-validator
The part I liked about express-validator is that it can be used to validate body/query/param together instead of having our own utility to extract content.

I think one of the issues may be validating arrays.
https://github.com/ratiw/Validator/issues/5